app/src/org/commcare/google/services/analytics/FirebaseAnalyticsUtil.java

```diff
@@ -219,4 +251,50 @@ public static boolean versionIncompatible() {
     private static boolean rateLimitReporting(double percentOfEventsToReport) {
         return Math.random() < percentOfEventsToReport;
     }
+
+    public static void reportVideoPlayEvent(String videoName, long videoDuration, long videoStartTime) {
+        String videoUsage;
+        if (videoDuration == -1) {
+            videoUsage = VIDEO_USAGE_OTHER;
```

Comment:
It would likely be useful to know how often this is happening, since we seem to swallow the signal otherwise. Worth adding `VIDEO_USAGE_UKNOWN_LENGTH` or similar?

Reply:
implemented 👍